src/views/bouquets/BouquetEditView.vue

```diff
@@ -69,49 +255,295 @@ onMounted(() => {
+            />
+            <DsfrInput
+              v-model="raison"
+              class="fr-mb-3w"
+              label="Raison d'utilisation dans ce bouquet*"
+              hint="Indiquer l'utilité de la donnée pour ce bouquet"
+              :label-visible="true"
+              :is-textarea="true"
+            />
+          </div>
+        </div>
+        <hr />
+        <div class="fr-grid-row align-center fr-mt-3w">
+          <div class="fr-col-12 fr-col-lg-5">
+            <Multiselect
+              ref="selector"
+              v-model="selectedDataset.id"
+              no-options-text="Précisez ou élargissez votre recherche"
+              placeholder="Rechercher une donnée dans Ecosphères"
+              name="select-datasets"
+              :clear-on-select="true"
+              :filter-results="false"
+              :min-chars="1"
+              :resolve-on-load="false"
+              :delay="0"
+              :searchable="true"
+              :options="search"
+            />
+          </div>
+        </div>
 
-          <Tooltip
-            title="Objectif du bouquet"
-            name="tooltip__objectif"
-            text="Ajoutez ici l'ensemble des informations nécessaires à la compréhension, l'objectif et l'utilisation du bouquet. N'hésitez pas à indiquer la réglementation ou une documentation liée au bouquet."
-          />
-          <Tooltip
-            title="Utilisez du markdown pour mettre en forme votre texte"
-            name="tooltip__markdown"
-            text="* simple astérisque pour italique *<br/> ** double astérisque pour gras **<br/> # un dièse pour titre 1<br/> ## deux dièses pour titre 2<br/> *  astérisque pour une liste<br/> lien : [[https://exemple.fr]]"
+        <h3 class="fr-mb-2w fr-mt-3w">
+          Vous ne trouvez pas la donnée dans Écosphères ?
+        </h3>
+        <div class="fr-grid-row align-baseline fr-mb-4w">
+          <div class="fr-col-12 fr-col-lg-5">
+            <DsfrInput
+              v-model="urlData"
+              placeholder="Url vers le jeu de données souhaité"
+              :label-visible="true"
+              class="fr-mb-md-1w"
+            />
+          </div>
+          <div class="fr-col-12 fr-col-lg-4 fr-ml-md-10w fr-mt-5w fr-m-lg-0">
+            <DsfrButton
+              label="Ajouter la donnée"
+              :secondary="true"
+              @click.prevent="addDatasetPropertysToExtras()"
+            />
+          </div>
+        </div>
+        <hr />
+
+        <h3>
+          Données sélectionnées
+          <span v-if="isDataProperty"
+            >({{ isDataProperty }})</span
+          >
+        </h3>
+        <div v-if="!isDataProperty" class="no-dataset fr-py-2 fr-px-3w">
+          <p class="fr-m-0">Aucune donnée ajoutée</p>
+        </div>
+        <div v-else>
+          <DsfrAccordionsGroup>
+            <li
+              v-for="property in datasetProperty[
+                `${config.universe.name}:datasets_properties`
```

Comment:
Pourquoi pas `getDataProperty` ?

Reply:
Je pense que le nom `DatasetProperties` est, au moins por l'instant, le plus parleur (on parle des métadonnées de la relation entre un bouquet et un dataset). Puisque la relation entre bouquet et dataset est 1 to many, un collection donc, je trouve que c'est lisible de parler de _datasets_properties_. Si cela te va.

Reply:
On a N properties pour un dataset mais on a aussi M datasets dans ce qui est actuellement `datasetProperty` (et qui devrait être `datasetsProperties`, cf https://github.com/ecolabdata/ecospheres-front/pull/110#discussion_r1386960380).

Le problème des nommages actuel se constate au niveau du `v-for="property in datasetProperty[...]` : 
- `property` contient les N properties d'un dataset donné (on le voit juste après avec `property.id`, `property.raison`, ...), donc devrait s'appeler `datasetProperties`
- `datasetProperty` contient les `datasetProperties` des M datasets, donc je vois pas vraiment d'autre nom que `datasetsProperties`.

Pour résumer : `v-for="property in datasetProperty[...]` -> `v-for="datasetProperties in datasetsProperties[...]`

Reply:
J'ai répondu à ta question dans ce thread, mais ma question initiale ne portait pas là dessus :

> Pourquoi pas `getDataProperty` ?

On utilise ici `${config.universe.name}:datasets_properties` alors que `const getDataProperty` vaut la même chose.

Reply:
Effectivement, je l'ai rajouté